examples/with-next-ui/package.json

```
@@ -0,0 +1,24 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "analyze": "cross-env ANALYZE=true next build"
```

```suggestion
    "start": "next start"
```

https://github.com/vercel/next.js/pull/38209/files#r916716116

---

Thanks for your suggestion, I have removed some extraneous code.